src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
It doesn't look like there's a standard way to do apidoc for websockets (just submitted a [question](https://github.com/apidoc/apidoc/issues/501) on the apidoc repo). Not sure what we should do for now but we should definitely document this API...